include/exadg/time_integration/time_int_gen_alpha_base.cpp

```diff
@@ -122,6 +130,23 @@ TimeIntGenAlphaBase<Number>::compute_const_vector(VectorType &       const_vecto
   const_vector.add(factor_acc, acceleration_n);
 }
 
+template<typename Number>
+void
+TimeIntGenAlphaBase<Number>::compute_const_vector_velocity_remainder(
+  VectorType &       const_vector,
+  VectorType const & displacement_n,
+  VectorType const & velocity_n,
+  VectorType const & acceleration_n) const
+{
+  double const factor_dis = -get_scaling_factor_mass_from_velocity();
+  double const factor_vel = alpha_f - (1.0 - alpha_f) * (gamma - beta) / beta;
+  double const factor_acc = -(1.0 - alpha_f) * (gamma - 2.0 * beta) / (2.0 * beta) * time_step;
+
+  const_vector.equ(factor_dis, displacement_n);
+  const_vector.add(factor_vel, velocity_n);
+  const_vector.add(factor_acc, acceleration_n);
+}
```

Comment:
we rewrite the velocity term from the gen.-alpha scheme in terms of the velocity. d^{n+1} is our unknown and the remaining terms are collected on the rhs. This can be compared to the remaining terms from the acceleration rewritten in terms of the displacement d.